openpype/plugins/publish/extract_review.py

```diff
@@ -788,7 +786,9 @@ def ffmpeg_full_args(
                     audio_filters.append(arg)
 
         all_args = []
-        all_args.append(path_to_subprocess_arg(self.ffmpeg_path))
+        all_args.append(
```

Comment:
This could be:
```py
all_args = [
    subprocess.list2cmdline(get_ffmpeg_tool_args("ffmpeg")),
    *input_args
]
```

Reply:
I think tuple expanding into list did not work for some DCCs. I'm not saying it is still an issue, but I'm scared to change it.